charmcraft.yaml

```diff
@@ -29,3 +31,10 @@ parts:
     prime:
       - charm_version
       - workload_version
+    override-pull: |
+      craftctl default
+      if [[ ! -f requirements.txt ]]
+      then
+          echo 'ERROR: Use "tox run -e build-dev" instead of calling "charmcraft pack" directly' >&2
+          exit 1
+      fi
```

Comment:
Do we still need that part since we export locally to a `requirements.txt` file?

Reply:
This will be updated in the next PR where I update charmcraft.yaml + remove the build wrapper :)